assets/queries/openAPI/success_response_code_undefined_get_operation/query.rego

```diff
@@ -0,0 +1,22 @@
+package Cx
+
+import data.generic.openapi as openapi_lib
+
+CxPolicy[result] {
+	doc := input.document[i]
+	openapi_lib.check_openapi(doc) != "undefined"
+	response := doc.paths[n][oper].responses
+
+	oper == "get"
+
+	object.get(response, "200", "undefined") == "undefined"
+	object.get(response, "202", "undefined") == "undefined"
+
+	result := {
+		"documentId": doc.id,
+		"searchKey": sprintf("paths.{{%s}}.{{%s}}.responses", [n, oper]),
```

Comment:
I think could be `sprintf("paths.{{%s}}.get.responses", [n])`, since `oper` will always be get, no?